Dockerfile.web

```diff
@@ -9,10 +9,10 @@ ARG NODE_OPTIONS="--v8-pool-size=4"
 # Pull from local registry - we can't pull from docker due to limits.
 # see https://catalog.redhat.com/software/containers/ubi9/nodejs-20/64770ac7a835530172eee6a9 for
 # details
-FROM registry.access.redhat.com/ubi9/nodejs-20 as static
-
+FROM node:20-alpine AS static
```

Comment:
Is this required? Will the web still build with the old base image from redhat?

Reply:
Nope. There's just something about the redhat node images. I've tried a lot of things. The error is "Dynamic require of "util" is not supported". And googling it is not very helpful. So we can't use docker bc we don't have a paid account and since it's not cached we will exceed rate limit? It seems like somehow yarn/corepack is part of the problem. Want to switch to npm instead of yarn? I don't have a lot more ideas.

Reply:
All good just wanted to clarify. Let's keep this and adjust if we run into rate limits.